recipes/default.rb

```diff
@@ -132,7 +132,7 @@
 end
 
 execute 'extract librenms archive' do
-  command "/usr/bin/unzip -o #{librenms_archive} -d #{librenms_rootdir}"
+  command "/usr/bin/unzip -o #{librenms_archive} -d #{librenms_rootdir} && chown #{librenms_username}:#{librenms_group} #{librenms_rootdir}"
```

Comment:
Since you want to make sure the rootdir always belongs to the right user and group, I think the following declaration is more suited, don't you think?

```
file librenms_rootdir do
  user librenms_username
  group librenms_group
end
```

Reply:
The content of rootdir should be chown'ed to librenms_username, but for reason of symlink and unzip it has to be done after extraction, unless you have a better way.

Reply:
Have you looked at using the 'ark' cookbook? It may help in doing all of this.

Reply:
Using ark for the whole fetch-extract-install is on my TODO (https://github.com/criteo-cookbooks/librenms/projects/1) but as this is blocking for us (daily update and discover cron are failing because of permission issues), I decided to make a quick workaround first.